File: 0453/453.go
func minimumMovesToEqualArrayElemen(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumMovesToEqualArrayElemen(root.Left)
	r := minimumMovesToEqualArrayElemen(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 0453/453.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var minimumMovesToEqualArrayElemen = function(root) {
    if (!root) return 0;
    return 1 + Math.max(minimumMovesToEqualArrayElemen(root.left), minimumMovesToEqualArrayElemen(root.right));
};